cornice_swagger/swagger.py

```diff
@@ -104,11 +111,25 @@ def _build_paths(self, ignore=['head', 'options'], default_tags=None,
             path = self._extract_path_from_service(service, **kwargs)
 
             for method_name, view, args in service.definitions:
-                if method_name.lower() in ignore:
+
+                if method_name.lower() in ignore_methods:
                     continue
 
                 op = self._extract_operation_from_view(view, args, **kwargs)
 
+                if any(ctype in op.get('consumes', []) for ctype in ignore_ctypes):
+                    continue
+
+                # XXX: Swagger doesn't support different schemas for for a same method
+                # with different ctypes as cornice. If this happens, you may ignore one
```

Comment:
Nit: Instead of `as cornice`, I'd probably write `like cornice` or `like cornice does`, and actually I'd probably write something even more verbose like `, even though cornice does`.